edge-modules/MetricsCollector/Settings.cs

```diff
@@ -23,7 +23,7 @@ public class Settings
                     configuration.GetValue<string>("AzMonWorkspaceId"),
                     configuration.GetValue<string>("AzMonWorkspaceKey"),
                     configuration.GetValue<string>("AzMonLogType", "edgeHubMetrics"),
-                    configuration.GetValue<string>("MetricsEndpointsCSV", "http://edgeHub:9600/metrics"),
+                    configuration.GetValue<string>("MetricsEndpointsCSV", "http://edgeHub:9600/metrics,http://edgeAgent:9600/metrics"),
```

Comment:
Should this default be edgehub and edgeagent or just empty? The settings json specifies the endpoints to scrape

Reply:
I tried earlier to just use empty because it should take its values from the settings.json, however they did not get read. I'm planning on asking Philip about this.

Reply:
The convention in other modules though is to specify in both places, which I disagree with as it is unnecessary.